tests/http_validator.py

```diff
@@ -306,6 +307,13 @@ def rate(org_domain, result_dict, global_translation, local_translation):
             org_www_domain,
             domain,
             should_create_recommendation)
+        rating += rate_sri(
```

Comment:
@cockroacher I get that SRI and CSP are close, but should SRI be running when "tests.http.csp-only" is set?

Reply:
@7h3Rabbit wanted to get input regarding SRI and CSP but was not interested in TLS and more.